sdk/eventhub/Microsoft.Azure.EventHubs/perf/Microsoft.Azure.EventHubs.Perf.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" />
     <PackageReference Include="Microsoft.Azure.Storage.Blob" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime" />
```

Comment:
This should no longer be needed.   The changes to move to the T2 management library were merged.   If you rebase, these refs should change.

Reply:
My bad.  This is the T1 project.